src/main/java/com/boggle/BoggleBoardGenerator.java

```diff
@@ -7,20 +7,20 @@
 import com.google.protobuf.ByteString;
 
 import javax.imageio.ImageIO;
-import javax.swing.text.html.parser.Entity;
 import java.awt.image.BufferedImage;
 import java.awt.image.DataBufferByte;
 import java.awt.image.WritableRaster;
 import java.io.File;
 import java.io.IOException;
-import java.lang.reflect.Array;
 import java.nio.file.Files;
 import java.nio.file.Path;
 import java.nio.file.Paths;
 import java.util.*;
 
 public class BoggleBoardGenerator {
-    static Map<Vertex,Character> map = new HashMap<>();
+    static Map<Vertex, Character> map = new HashMap<>();
+    static Map<String,Character> alreadyProcessedVertices = new HashMap<>();
+    static Map<Vertex, Character> finalMap = new HashMap<>();
```

Comment:
why did you make these static variables? bc intellij told you to?

Reply:
a static variable is something that has a lifetime over all the classes instance, so imagine if you called checkVertexRepetition the second time it would use all these maps with filled in values

Reply:
i didn't make it static - i guess that happened automatically, i changed it back.